addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -179,9 +179,17 @@ registerWebsitePreviewTour("megamenu_active_nav_link", {
         run: "click",
     },
     {
-        content: "Edit link",
+        content: "Click 'edit link' button if URL input is now shown",
         trigger: "#create-link",
-        run: "click",
+        run(actions) {
+            // Note: the 'create-link' button is always here, however the input for the URL might not be
+            // We have to consider both cases:
+            // 1. Single-app website build: a few menu, so no extra menu added and the URL input is shown
+            // 2. Multi-app website build:  many menu, so extra menu added and the URL input is not shown
+            if (!(document.querySelector("#o_link_dialog_url_input"))) {
```

Comment:
not really sure using `document` is the best approach, `actions.anchor` is the "edit link" button so can't be used directly for querySelector-ing (it will return undefined)